src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -117,6 +119,20 @@ class GrabbitConfiguration {
         )
     }
 
+    /**
+     * Sorts the {@code pathConfigurations} on the basis of path depth
+     *
+     * @param pathConfigurations
+     *          Collection of {@link PathConfiguration} to sort
+     * @return
+     *          Sorted Collection of {@link PathConfiguration}
+     */
+    private static Collection<PathConfiguration> sortPathConfigByDepth(Collection<PathConfiguration> pathConfigurations){
```

Comment:
Generally not a good idea to mutate an argument like this. In addition, we are mutating the argument, returning it; then throwing away the return above.